src/constants.ts

```diff
@@ -63,5 +63,9 @@ export const EXTERNAL_WHITELIST_PROVIDERS = [
     height: '398px',
   },
   { name: 'MolView', url: ['embed.molview.org'] },
```

Comment:
Ser riktig ut for meg, men er ikke kjent med hva disse brukes til.

Reply:
Dette er whitelisting av urler som kan benyttes som embeds. statisk har med urler fordi det er en iframe-embed og ikkje en external-embed som bruker oembed.

Reply:
lgtm 🤷